pallets/creditcoin/src/lib.rs

```diff
@@ -597,12 +642,49 @@ pub mod pallet {
 					verify_result,
 					on_success,
 					on_failure,
-					transfer_status,
+					status,
 					&u_t as &dyn core::fmt::Debug,
 				);
 			});
 
-			u_transfer.for_each(drop);
+			let u_collect_coins =
+				UnverifiedCollectCoins::<T>::iter().map(|(deadline, id, u_cc)| {
+					let storage_key = (deadline, id.clone()).encode();
+					let status = ocw::LocalVerificationStatus::new(&storage_key);
+					if status.is_complete() {
+						log::debug!("Already handled CollectCoins ({:?}, {:?})", deadline, id);
+						return;
+					}
+					let mut collect_coins = None;
+					let verify_result = Self::verify_cc_ocw(&u_cc, &mut collect_coins);
```

Comment:
why mutate `collect_coins` instead of just returning it from `verify_cc_ocw`?

Reply:
It is confusing if a method involving verification or validation were to return state not related to the verification/validation itself.